### ISSUE TITLE
package supplemental model data

### ISSUE THREAD

**Issue by elbashandy** [LBL-EESA/TECA#317]
- why is it so big?
-- how many parameters are there and their type?
- investigate Deeplabv3Plus's model size

**Comment by burlen** [comment]
clarifying the details:

* we want to deploy the new detector, but some supplemental data needs to be packaged as well. the smaller this data is the better.
* in terms of deploying the algorithm we only need to be able to do inference, not training.
* we can't include a 200MB file directly in the repo. having the supplemental data stored elsewhere complicates automated installs for example pip. not to mention that when run in parallel this data needs to be moved to all MPI ranks. 
* we can't put the data on the www and and pull it at run time given that compute nodes won't generally have access to outside networks.

the goal of this task is to be sure that

1. we are packaging this data in the best possible way
2. nothing extra (ie not needed for inference) is packaged in the supplemental data
3. nothing extra is included in the nested deeplab v3 model
3. if there is nothing that can be done to reduce the size, can we use compression to reduce the size?

It is worth asking various experts about:
1. the expected size of the supplemental data needed for inference.
2. what steps can be taken to minimize the size
3. what is the optimal way to package and deploy this data for inference.

[1] - https://discuss.pytorch.org/
[2] - https://github.com/sksq96/pytorch-summary
[3] - https://stackoverflow.com/questions/42703500/best-way-to-save-a-trained-model-in-pytorch

**Comment by burlen** [comment]
This issue is being replaced by #358
